lib/transformations/resolve/resolve.js

```diff
@@ -0,0 +1,46 @@
+// Press ctrl+space for code completion
+export default function transformer(file, api) {
```

Comment:
@pksjce  is going to add a wrapper using recast to make it easier to contribute or add packages. We should use recast instead of jscodeshift eventually.